mobile/lib/ui/viewer/actions/file_selection_actions_widget.dart

```diff
@@ -646,6 +691,8 @@ class _FileSelectionActionsWidgetState
     );
     if (actionResult?.action != null) {
       if (actionResult!.action == ButtonAction.first) {
+        generatedPathNotifier.value =
```

Comment:
I don't see why we need to use a `ValueNotifier` here. We aren't listening to it anywhere. Can just use a String or String? if nullable.